### ISSUE TITLE
Error on Login

### ISSUE THREAD

**Issue by Lanzlo** [Lethay/TalentedClassic#31]
Date: 2019-11-14 14:15:26
ID: 1
Error occured in: Global
Count: 1
Message: ..\AddOns\Talented\editmode.lua line 108:
   bad argument #1 to 'ipairs' (table expected, got nil)
Debug:
   [C]: ?
   Talented\editmode.lua:108: ValidateTemplate()
   Talented\core.lua:355: LoadTemplates()
   Talented\core.lua:73:
      Talented\core.lua:70
   [C]: ?
   ...face\AddOns\Atlas\Libs\AceAddon-3.0\AceAddon-3.0.lua:70:
      ...face\AddOns\Atlas\Libs\AceAddon-3.0\AceAddon-3.0.lua:65
   ...face\AddOns\Atlas\Libs\AceAddon-3.0\AceAddon-3.0.lua:498: InitializeAddon()
   ...face\AddOns\Atlas\Libs\AceAddon-3.0\AceAddon-3.0.lua:613:
      ...face\AddOns\Atlas\Libs\AceAddon-3.0\AceAddon-3.0.lua:605
Locals:
None
AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AdvancedInterfaceOptions, v1.3.3
  Atlas, vv1.45.01.11302
  AtlasClassicWoW, vv1.45.00.11302.05
  AtlasLootClassic, vv1.3.4
  AtlasLootClassicData, vv1.3.4
  AtlasLootClassicDungeonsAndRaids, vv1.3.4
  AtlasLootClassicMaps, vv1.0.0
  AtlasQuest, v4.11.57
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  auccheapsearcher, v20191013-retail
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  BeanCounter, v8.2.6434 (SwimmingSeadragon)
  Bejeweled, v1.13.2
  CallbackHandler10, v
  ClassicQuestLog, v1.4.5-Classic
  ClassicAuraDurations, v
  ClassicSpellActivations, v
  ClassicThreatMeter, v1.10
  Clique, vv1.0.3-classic
  DBMCore, v1.13.18
  DBMDefaultSkin, v
  DBMRaidLeadTools, vr124
  DBMSMGEventsPack, v1.5.3
  DBMStatusBarTimers, v
  DejaClassicStats, v1302r037
  Enchantrix, v8.2.6428 (SwimmingSeadragon)
  EnchantrixBarker, v8.2.6469 (SwimmingSeadragon)
  ExtTabardUI, v1.0.0
  GSE, v2.4.23
  GSEGUI, v2.4.23
  GSELDB, v2.4.23
  GuildRosterManager, v1.831
  KuiMedia, v
  KuiNameplates, v
  KuiNameplatesCore, v
  LibStub, v
  Necrosis, v5.2
  NugEnergy, v1.13.11
  OmniCC, v8.2.5
  QuestFrameFixer, v1.1.1
  QuestIconDesaturation, v3.1
  Questie, v5.2.1
  RCLootCouncilClassic, v0.3.1
  RealMobHealth, v2.13
  SilverDragon, vv11302.0
  SilverDragonClassic, vv11302.0
  SimplyGInv, v1.13.2
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  Stubby, v8.2.6376 (SwimmingSeadragon)
  Talented, vv191112
  BlizRuntimeLib_enUS v1.13.2.11302 <none>
  (ck=6ea)

**Comment by Lethay** [comment]
Do you not have Talented_Data installed and activated?

**Comment by Lethay** [comment]
Ah, I looked at one of your other errors and you do. I tried playing with the latest version for a little bit on a couple of characters to myself but I don't get this error, and I'm not sure what might be causing it for you.
The line causing the error is checking the task of the active template, but the class is found not to exist. The active template is read from the database of all templates. In other words, one of the templates in the database is invalid for some reason. Or, so it seems.
